internal/manifest/os.go

```diff
@@ -537,6 +537,16 @@ func (p *OS) serialize() osbuild.Pipeline {
 	}
 
 	if p.OSTreeRef != "" {
+		pipeline.AddStage(osbuild.NewSystemdJournaldStage(
```

Comment:
@achilleas-k  Seems like a straight forward change to me, is there any change with the manifests regeneration go run ./cmd/gen-manifests -output test/data/manifests -workers 50 -images edge-commit,edge-container, as
the tests are failing.